### PR TITLE
Ci-build-me bot works, fixes #4988 

### LINKED ISSUE #4988: Buildkite CI: Solve non-core member CI running security issue
We want Buildkite jobs to appear on PRs in the same way that circle jobs appear on PRs automatically.

For security reasons, we don't want to run arbitrary code in a PR from someone who is not on the core team. We need to make sure to detect this.

See a comment below for the latest proposal.

Historical proposals:

Proposals:
1. Hit GitHub API, if PR is from a fork; deploy a small github bot that listens to a special comment from a core team member (example "!start-test") then allow the test to run.
2. Hit GitHub API to pull core team from org, if PR author is not in core team; use a https://buildkite.com/docs/pipelines/block-step until a core team member unblocks.

(1) This is out-in-the-open and mirrors other popular open source projects -- like Rust and Swift. But it requires us to maintain a GitHub bot -- though we can do this with a lambda/cloud-function that listens to webhook events on comments.
(2) feels like it's more holistic in the buildkite world, but it's a little more behind-the-scenes and may be opaque to external contributors. Additionally, Brandon's jsonschema-to-dhall library chokes on the block-step schema, so we'd need to update that.

Of these two choices, (1) seems better to me. Feel free to add more ideas in the comments below also.

Changelog:
* Add historical proposals and note the newest is at the bottom
* Change (1) to if PR is from a fork

### PR DESCRIPTION
**Pull request by bkase**
Created a bot to fix #4988 

See [README](frontend/ci-build-me/README.md) for details on how to test and deploy.

Tested by applying the `ci-build-me` label to a PR, observing a buildkite job starts. Also, tried pushing to a PR with the label on it and a buildkite job starts. Importantly, pushing to a PR without the label does not trigger a buildkite job. I manually verified that this only works for PRs opened from internal PRs. See https://github.com/CodaProtocol/coda/pull/5233 .

Additionally, a comment of exactly `!ci-build-me` on a pull-request, if that comment is authored by a user that is part of the CodaProtocol organization will also trigger a build. I verified that folks who are not part of the organization, cannot trigger builds with this special comment as well (see the comment via @mobileink ).

The secrets needed to test and deploy this bot are stored in AWS secrets manager under `coda/ci-build-me-bot`.

This bot is currently live on Google Cloud Functions.

### CODE REVIEW

**Comment by bkase** [comment]
> Given the vulnerability this opens up if someone does not toggle correctly (which seems like a very easy mistake to make), I think we should wait to merge this until we have a better approach. Perhaps switching over to requiring comments so that it's tied to a specific sha, or a different plan.

Alright fair, I'll make the label only work for core team and require comment triggers for external contributors. Shouldn't be too hard.

**Comment by bkase** [comment]
testing

**Comment by bkase** [comment]
!ci-build-me

**Comment by bkase** [comment]
!ci-build-me

**Comment by bkase** [comment]
!ci-build-me

**Comment by bkase** [comment]
!ci-build-me

**Comment by bkase** [comment]
!ci-build-me

File: frontend/ci-build-me/package.json [hunk 1]
```diff
@@ -0,0 +1,53 @@
+{
+  "name": "github-webhook-cloud-function",
+  "version": "1.0.0",
+  "description": "Update Trello from GitHub Webhooks using Google Cloud Functions",
```
Comment (Schmavery) [inline frontend/ci-build-me/package.json:4]: small thing -- description is wrong
edit: actually much of the metadata in this package.json isn't updated for this project

**Pull request merged**